.github/workflows/code-quality.yml

```
@@ -9,15 +9,19 @@ on:
   push:
     branches:
       - master
+    paths:
+      - "pyproject.toml"
   pull_request:
+    paths:
+      - "pyproject.toml"
```

You can remove this to run this action every time.
```suggestion
  pull_request:
```